migrations/1700850247612-DepositsMv.ts

```diff
@@ -0,0 +1,86 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class DepositsMv1700850247612 implements MigrationInterface {
+  name = "DepositsMv1700850247612";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP MATERIALIZED VIEW IF EXISTS "deposits_mv"`);
+    await queryRunner.query(`DROP VIEW IF EXISTS "deposits_filtered_referrals"`);
+    await queryRunner.query(`CREATE VIEW "deposits_filtered_referrals" AS 
+      SELECT 
+        d.id,
+        d."depositorAddr",
+        d."stickyReferralAddress",
+        d."depositDate",
+        d."priceId",
+        d."tokenId",
+        d.amount,
+        d."rewardsWindowIndex",
+        case when d."rewardsWindowIndex" = c."windowIndex" then d."rewardsWindowIndex" else -1 end as "referralClaimedWindowIndex",
+        hmp."usd",
+        t.decimals
+      FROM deposit d
+      JOIN historic_market_price hmp on d."priceId" = hmp.id
+      JOIN token t on d."tokenId" = t.id
+      LEFT JOIN claim c on d."rewardsWindowIndex" = c."windowIndex" and d."stickyReferralAddress" = c."account"
+      WHERE "stickyReferralAddress" is not null
+        AND "depositDate" is not null
+        AND "tokenId" is not null
+        AND "priceId" is not null
+        AND status = 'filled'
+        AND d."acxUsdPrice" is not null
+        AND (
+          d."depositDate" < '2023-11-30 20:00:00' 
+          OR (d."depositDate" >= '2023-11-30 20:00:00' AND d."destinationChainId" != 10)
+        );
+    `);
+    await queryRunner.query(`CREATE MATERIALIZED VIEW "deposits_mv" AS 
```

Comment:
The `MATERIALIZED VIEW` definition is the same, I just needed to DROP and CREATE it again